test/server/framework/framework.dart

```diff
@@ -0,0 +1,277 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
I'm not sure this repo needs the task framework used by the devicelab.

Reply:
Remove unnecessary utils.